### PR TITLE
[feature] Add configuration item to give path to NONMEM license file

### PR DESCRIPTION
**Pull request by pharmpy-dev-123**
Fixes https://github.com/pharmpy/pharmpy/issues/733.

  - [x] ~Escape path (what does nmfe expect when a file path contains spaces for instance?)~
  - [x] ~Test~

### CODE REVIEW

File: src/pharmpy/plugins/nonmem/run.py [hunk 1]
```diff
@@ -113,6 +112,18 @@ def nmfe_path():
     return str(path)
 
 
+def nmfe(*args):
+    conf_args = []
+    if conf.licfile is not None:
+        conf_args.extend(['-licfile', str(conf.licfile)])
```
Comment (rikardn) [inline src/pharmpy/plugins/nonmem/run.py:118]: I think we need `-lcfile=...`

nmfe documentation:

```
Usage:
nmfe74 infile outfile [-background] [-PARAFILE=file.pnm] [-licfile=filename]
  [-prsame] [-prdefault] [-prcompile] [-tprdefault] [-trskip] [-nobuild] 
  [-maxlim=0, 1, 2, or 3]  [-locfile=file] 
  [-rundir=dirname]
  [-runpdir=dirname] [-nmexec=filename] [-xmloff]
  [-parafprint=0-99999] [-simparon] [-flushtime>=0.0] 
Default:
nmfe74 infile outfile
  -licfile=/opt/nm744/license/nonmem.lic
  -prsame (if possible; else -prcompile)
  -rundir=/home/rikard
  -runpdir=temp_dir -nmexec=nonmem
```

**Review by rikardn — APPROVED** [review]
This looks good. I think we can ignore spaces in paths. Do you want to squash or should I squash and merge?

**Comment by pharmpy-dev-123** [comment]
> This looks good. I think we can ignore spaces in paths. Do you want to squash or should I squash and merge?

OK. I just squashed. Should be good to merge.

**Comment by rikardn** [comment]
Thanks!

**Pull request merged**